build.cake

```diff
@@ -153,6 +153,30 @@ Task("Zip-Files")
     Zip(binDir, packagePath, files);
 });
 
+Task("Create-Chocolatey-Packages")
+    .IsDependentOn("Copy-Files")
+    .WithCriteria(() => isRunningOnWindows)
+    .Does(() =>
+{
+  // Create Cake package.
+  ChocolateyPack("./nuspec/Cake.Portable.nuspec", new ChocolateyPackSettings {
+      Version = semVersion,
+      Files = new [] {
+        new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.exe")},
```

Comment:
This is a bug. I suspect the `FilePath` (absoluteBinDir) and the `ConvertableFilePath` (Cake.exe) converts to string to satisfy the `+` (since they both have implicit conversion to string), and since FilePath has implicit conversion from string, the result gets wrong. Need to think about this one...